server/build/index.js

```diff
@@ -7,6 +8,13 @@ export default async function build (dir) {
     clean(dir)
   ])
 
+  await Promise.all([
+    runCompiler(compiler),
+    babel(dir)
```

Comment:
We have to do a specific babel compilation instead of using `webpack` and `babel-loader` for `pages/_document.js` since the file is used only on server side.